.travis.yml

```diff
@@ -10,4 +13,7 @@ before_install:
  - git submodule update --init --recursive
  
 scala:
-   - 2.11.8
\ No newline at end of file
+  - 2.11.12
```

Comment:
While we're updating things, do we want to consider targeting Scala 2.12 or 2.13? 2.12 improved a lot of things under the hood to use the JVM much more effectively, and 2.13 improved a lot of the built in collections. I think both should still work with JDK 11.

Reply:
The reason I've left it at 2.11 is because that's what Spark is currently built with--the latest stable version of Spark (v2.4.5 released in Feb. 2020) was built with 2.11. So, unless we Spark build from source, I think it makes more sense to use the same version of Scala that the pre-built package uses. Would you agree or no?

Reply:
I agree, let's stick to the same version Spark is using.